swig/python3/tests/NotificationTest.py

```diff
@@ -0,0 +1,570 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+__author__ = "Rastislav Szabo <raszabo@cisco.com>, Lukas Macko <lmacko@cisco.com>"
+__copyright__ = "Copyright 2016, Cisco Systems, Inc."
+__license__ = "Apache 2.0"
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# sysrepod and notifications_test_app must be in PATH
+
+from ConcurrentHelpers import *
+from random import randint
+import signal
+import os
+import subprocess
+import TestModule
+import libsysrepoPython3 as sr
+
+
+class NotificationTester(SysrepoTester):
+
+    def cleanup(self):
+        if self.filename and os.path.isfile(self.filename):
+            os.unlink(self.filename)
+
+    def subscribeStep(self, xpath):
+        self.filename = "notifications_test_" + str(randint(0, 9999))
+        self.process = subprocess.Popen(["notifications_test_app", xpath, self.filename])
```

Comment:
I think we agreed yesterday that this external app will be replaced by python (binding-native) code, correct me if I am wrong.